frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -7,7 +7,7 @@ import styles from './styles.module.scss';
 
 type Properties = {
   iconName: IconName;
-  onClick?: () => void;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
I am not sure if we need to deal with mouse event for this button
```suggestion
  onClick?: () => void;
```

Reply:
I use it in the ArticleView component to check the name of current button in onButtonClick callback. As an alternative, I can make separate callbacks for different icon buttons, is it better?

Reply:
There is a comment about it https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/173#discussion_r1318333698